AirBench/Controllers/ReviewController.cs

```diff
@@ -0,0 +1,58 @@
+﻿using AirBench.Data;
+using AirBench.FormModels;
+using AirBench.Models;
+using AirBench.Repositories;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using System.Web.Mvc;
+
+namespace AirBench.Controllers
+{
+    public class ReviewController : Controller
+    {
+        private Context context;
+
+        public ReviewController()
+        {
+            context = new Context();
+        }
+
+        // GET: Review
+        public ActionResult Index()
+        {
+            return View();
+        }
+
+        [Authorize]
+        [HttpGet]
+        public ActionResult Create()
```

Comment:
Should this action method include an `id` parameter for the bench to add a review to? That'd also allow you to retrieve the bench information for display on the view.